readme.md

```diff
@@ -55,20 +56,110 @@ function App() {
 ```ts
 import { useSendTransaction, useWriteContract } from "@arcana/ca-wagmi"
 
-// has same API as wagmi `useSendTransaction`
-const { sendTransaction } = useSendTransaction() 
+// has same API as wagmi `useSendTransaction` and `useSendTransactionAsync`
+const { sendTransaction, sendTransactionAsync } = useSendTransaction() 
 
-// has same API as wagmi `useWriteContract`
-const { writeContract } = useWriteContract() 
+// has same API as wagmi `useWriteContract` and `useWriteContractAsync`
+const { writeContract, writeContractAsync } = useWriteContract() 
 ```
 
-### useBalance
+### Arcana hooks
 
-```ts
-import { useBalance } from "@arcana/ca-wagmi"
+- `useBalance({ symbol: string })`
```

Comment:
@Abhishek, why aren't we using #### for `useBalance(...` heading? It will show up nicely in TOC for this page. bullet list wont show up in TOC.

It is easy to cross reference TOC headings in docs.

Reply:
Ok, will update with other suggestions also. Also there is a mixing of type and data type - will change to type everywhere.